kubernetes/linux/setup.sh

```diff
@@ -61,7 +61,7 @@ sudo tdnf install jq-1.6-1.cm2 -y
 #used to setcaps for ruby process to read /proc/env
 sudo tdnf install libcap -y
 
-sudo tdnf install telegraf-1.27.2 -y
+sudo tdnf install telegraf-1.27.3 -y
```

Comment:
@jatakiajanvi12 , are we not seeing any functionality issues with this update, right? and also telegraf failed to connect port:25226/25228 errors are not observed this version, right? did you verify that, applying the prometheus configmap (node & cluster) level, and no errors? Reading release notes - https://github.com/influxdata/telegraf/releases/tag/v1.27.3 and looks like  only 2 changes related to input.prometheus plugin which seems safe changes but  quick functionality test with this. We would need to monitor CI/CD clusters after we toggle release candidate image.

Reply:
Yes have validated this on a cluser.Dont see the port errors and the metrics seem to be working